docker/Dockerfile-tests

```diff
@@ -0,0 +1,38 @@
+FROM fedora:28
+
+RUN dnf -y install \
+  --setopt=deltarpm=0 \
+  --setopt=install_weak_deps=false \
+  --setopt=tsflags=nodocs \
+  bash \
+  nmap-ncat \
+  python2-devel \
+  python3-devel \
+  # Install RPM dependencies from estuary-api otherwise they'll be installed via pip which won't map
+  # to production
+  python2-fedmsg \
+  python3-fedmsg \
+  python3-flake8 \
+  python2-flask \
+  python3-flask \
+  python2-mock \
+  python3-mock \
+  # Until 1.6.0 is supported, we need to hardcode these two entries
+  python2-neo4j-driver-1.5.3 \
+  python3-neo4j-driver-1.5.3 \
```

Comment:
Once the [PR](https://github.com/release-engineering/estuary-api/pull/138) in estuary-API is merged, we can remove these right?

Reply:
It turns out neomodel doesn't support python-neo4j-driver 1.6. I filed a PR upstream to fix that.